This patch fixes petbuild JWM-2.3.7 crash report #3073
reported by 'gychang' 
https://forum.puppylinux.com/viewtopic.php?p=56878#p56878
fix from jwm by author 'rdnvdr' - Andrey Rodionov 20170930
commit bc58de37452d843e6e2e50b4c67be1f1585a6ccb
diff -rupN jwm-2.3.7.orig/src/move.c jwm-2.3.7/src/move.c
--- jwm-2.3.7.orig/src/move.c	2022-05-09 12:57:19.033983641 +1000
+++ jwm-2.3.7/src/move.c	2022-05-09 12:58:50.537984994 +1000
@@ -333,6 +333,7 @@ char MoveClientKeyboard(ClientNode *np)
    } else {
       height = np->height;
    }
+   currentClient = np;
 
    for(;;) {
 
